### PR TITLE
Created Reports Explorer

### PR DESCRIPTION
**Pull request by chalettu**
PV Ticket https://www.pivotaltracker.com/story/show/142087845

### CODE REVIEW

**Comment by chalettu** [comment]
@miq-bot add_label euwe/no
@miq-bot add_label enhancement

**Comment by chalettu** [comment]
@miq-bot remove_label wip

**Comment by chalettu** [comment]
report-explorer

**Comment by chalettu** [comment]
@miq-bot add_label reports

File: client/app/reports/reports-explorer.component.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  function resolveReports(limit, offset, refresh) {
+    if (!refresh) {
+      vm.loading = true;
+    }
+    vm.offset = String(offset);
+
+    ReportsService.getMinimal(vm.filters).then(setResultTotals);
+    ReportsService.getReports(limit, vm.offset, vm.filters, vm.sortConfig)
+      .then(querySuccess)
+      .catch(queryFailure);
```
Comment (AllenBW) [inline client/app/reports/reports-explorer.component.js]: this should be in the above response resolution:
`.then(querySuccess, queryFailure)`

Comment (chalettu) [inline client/app/reports/reports-explorer.component.js]: Added this change in.

**Comment by simaishi** [comment]
Fine backport details:

```
$ git log -1
commit ef6e3e798342ec82696aafedd8d39d4dd4853ee0
Author: Chris Kacerguis <chriskacerguis@users.noreply.github.com>
Date:   Wed Mar 29 09:16:19 2017 -0500

    Merge pull request #611 from chalettu/report-explorer
    
    Created Reports Explorer
    (cherry picked from commit ecb76f5e810fbd5d3031484602dfdc811e295608)
```

**Pull request merged**